app/forms/steps/capital/property_owner_fieldset_form.rb

```diff
@@ -14,6 +14,8 @@ class PropertyOwnerFieldsetForm < Steps::BaseFormObject
         in: RelationshipType.values.map(&:to_s).push(PropertyOwner::OTHER_RELATIONSHIP)
       }
 
+      validates_numericality_of :percentage_owned, greater_than_or_equal_to: 0.0, less_than_or_equal_to: 100.0
```

Comment:
Wondering if this should be just less than 100 - else the applicant wouldn't own it at all, right?